sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/src/main/java/io/sermant/registry/entity/GraceShutdownBehavior.java

```diff
@@ -56,6 +56,8 @@ public void run() {
     }
 
     private void graceShutDown() {
+        // wait notify consumer complete
+        CommonUtils.sleep(graceConfig.getWaitNotifyTime() * ConfigConstants.SEC_DELTA);
```

Comment:
It is recommended to set the unit of WaitNotifyTime to milliseconds, so that you do not need to convert to milliseconds every time

Reply:
number of millisecond is too big and not easy to count how many 0